data-protocols/ids/ids-api-multipart-dispatcher-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/dispatcher/sender/SenderDelegateContext.java

```diff
@@ -0,0 +1,57 @@
+/*
+ *  Copyright (c) 2022 Fraunhofer Institute for Software and Systems Engineering
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Fraunhofer Institute for Software and Systems Engineering - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.ids.api.multipart.dispatcher.sender;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.eclipse.dataspaceconnector.ids.spi.transform.IdsTransformerRegistry;
+import org.jetbrains.annotations.NotNull;
+
+import java.net.URI;
+
+/**
+ * Contains services and context information required by sender delegate classes.
+ */
+public class SenderDelegateContext {
+    private final URI connectorId;
+    private final ObjectMapper objectMapper;
+    private final IdsTransformerRegistry transformerRegistry;
+    private final String idsWebhookAddress;
+
+    public SenderDelegateContext(@NotNull URI connectorId,
```

Comment:
minor nit: the `@NotNull` is purely for IntelliJ's code inspection feature. Unless there is a `Objects.requireNotNull` assertion in code, nothing will in fact be checked at runtime.

Reply:
good call, added non-null checks